src/modules/hip/kernel/log.hpp

```diff
@@ -0,0 +1,114 @@
+#include <hip/hip_runtime.h>
+#include "rpp_hip_common.hpp"
+
+template <typename T>
+__device__ void log_hip_compute(T *srcPtr, d_float8 *src_f8, d_float8 *dst_f8)
+{
+    if constexpr (std::is_same<T, schar>::value)
+        rpp_hip_math_add8_const(src_f8, src_f8, (float4)128);
+
+    dst_f8->f1[0] = __logf(fabsf(src_f8->f1[0]));
```

Comment:
You can add this log compute function in rpp_hip_common.hpp so that it can be reused for other functions in future

Reply:
Sure